ERROR - 2023-09-04 22:33:08 --> Error connecting to the database: No connection could be made because the target machine actively refused it.

CRITICAL - 2023-09-04 22:33:08 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\jalan\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\jalan\system\Database\BaseBuilder.php(1588): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\jalan\app\Models\M_model.php(34): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 C:\xampp\htdocs\jalan\app\Controllers\Home.php(48): App\Models\M_model->getWhere2('user', Array)
#4 C:\xampp\htdocs\jalan\system\CodeIgniter.php(832): App\Controllers\Home->aksi_login()
#5 C:\xampp\htdocs\jalan\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2023-09-04 22:34:05 --> Error connecting to the database: No connection could be made because the target machine actively refused it.

CRITICAL - 2023-09-04 22:34:05 --> Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it.

#0 C:\xampp\htdocs\jalan\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\jalan\system\Database\BaseBuilder.php(1588): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\jalan\app\Models\M_model.php(34): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 C:\xampp\htdocs\jalan\app\Controllers\Home.php(48): App\Models\M_model->getWhere2('user', Array)
#4 C:\xampp\htdocs\jalan\system\CodeIgniter.php(832): App\Controllers\Home->aksi_login()
#5 C:\xampp\htdocs\jalan\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2023-09-04 22:36:35 --> Table 'dbjalan.user' doesn't exist in engine
CRITICAL - 2023-09-04 22:36:35 --> Call to a member function getRowArray() on bool
#0 C:\xampp\htdocs\jalan\app\Controllers\Home.php(48): App\Models\M_model->getWhere2('user', Array)
#1 C:\xampp\htdocs\jalan\system\CodeIgniter.php(832): App\Controllers\Home->aksi_login()
#2 C:\xampp\htdocs\jalan\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#3 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2023-09-04 22:38:15 --> Table 'dbjalan.user' doesn't exist in engine
CRITICAL - 2023-09-04 22:38:15 --> Call to a member function getRowArray() on bool
#0 C:\xampp\htdocs\jalan\app\Controllers\Home.php(48): App\Models\M_model->getWhere2('user', Array)
#1 C:\xampp\htdocs\jalan\system\CodeIgniter.php(832): App\Controllers\Home->aksi_login()
#2 C:\xampp\htdocs\jalan\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#3 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2023-09-04 22:43:05 --> syntax error, unexpected 'public' (T_PUBLIC), expecting end of file
#0 C:\xampp\htdocs\jalan\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('home/aksi_login')
#1 C:\xampp\htdocs\jalan\system\CodeIgniter.php(726): CodeIgniter\Router\Router->handle('home/aksi_login')
#2 C:\xampp\htdocs\jalan\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2023-09-04 22:43:20 --> syntax error, unexpected 'public' (T_PUBLIC), expecting end of file
#0 C:\xampp\htdocs\jalan\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('home')
#1 C:\xampp\htdocs\jalan\system\CodeIgniter.php(726): CodeIgniter\Router\Router->handle('home')
#2 C:\xampp\htdocs\jalan\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2023-09-04 22:43:35 --> syntax error, unexpected '}', expecting end of file
#0 C:\xampp\htdocs\jalan\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('home')
#1 C:\xampp\htdocs\jalan\system\CodeIgniter.php(726): CodeIgniter\Router\Router->handle('home')
#2 C:\xampp\htdocs\jalan\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2023-09-04 22:43:44 --> syntax error, unexpected 'public' (T_PUBLIC), expecting end of file
#0 C:\xampp\htdocs\jalan\system\Router\Router.php(175): CodeIgniter\Router\Router->autoRoute('home')
#1 C:\xampp\htdocs\jalan\system\CodeIgniter.php(726): CodeIgniter\Router\Router->handle('home')
#2 C:\xampp\htdocs\jalan\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2023-09-04 22:44:04 --> Table 'dbjalan.user' doesn't exist in engine
CRITICAL - 2023-09-04 22:44:04 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\jalan\app\Controllers\Home.php(17): App\Models\M_model->join2('user', 'level', 'user.level = le...')
#1 C:\xampp\htdocs\jalan\system\CodeIgniter.php(832): App\Controllers\Home->index()
#2 C:\xampp\htdocs\jalan\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#3 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2023-09-04 22:53:58 --> Error connecting to the database: Unknown database 'dbjalan'
CRITICAL - 2023-09-04 22:53:58 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'dbjalan'
#0 C:\xampp\htdocs\jalan\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\jalan\system\Database\BaseBuilder.php(1588): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\jalan\app\Models\M_model.php(34): CodeIgniter\Database\BaseBuilder->getWhere(Array)
#3 C:\xampp\htdocs\jalan\app\Controllers\Home.php(48): App\Models\M_model->getWhere2('user', Array)
#4 C:\xampp\htdocs\jalan\system\CodeIgniter.php(832): App\Controllers\Home->aksi_login()
#5 C:\xampp\htdocs\jalan\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#9 {main}
ERROR - 2023-09-04 22:54:51 --> Table 'dbjalan.user' doesn't exist in engine
CRITICAL - 2023-09-04 22:54:51 --> Call to a member function getRowArray() on bool
#0 C:\xampp\htdocs\jalan\app\Controllers\Home.php(48): App\Models\M_model->getWhere2('user', Array)
#1 C:\xampp\htdocs\jalan\system\CodeIgniter.php(832): App\Controllers\Home->aksi_login()
#2 C:\xampp\htdocs\jalan\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#3 C:\xampp\htdocs\jalan\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\jalan\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\jalan\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
